src/ProjectTemplates/Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-    "parser": "typescript-eslint-parser"
-}
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module"
+    }
```

Comment:
Not sure about this. Why did we add it?

Reply:
without this the import declarations fail. Something seems have changed in the dependencies